app/templates/editions/octane.hbs

```diff
@@ -4,26 +4,21 @@
 <section class="container" aria-labelledby="octane-edition">
   <h1 id="octane-edition">The Octane Edition of Ember</h1>
   <p class="intro">
-    It has arrived!
-    As of version 3.15 of Ember, when you generate a new app, you get everything you need to build your app using Octane.
-    You can read more about features and upgrade strategies in the
-    <a href="https://blog.emberjs.com/2019/12/20/octane-is-here.html">the official Ember blog post</a>.
+    Ember Octane is the latest edition in Ember.
+    An edition is a collection of features and patterns that represent
+    the best way to build Ember apps for a range of Ember versions.
+    Octane has modern, streamlined components and state management that make it fun
+    to build web applications.
+    JavaScript and HTML are at the front and center, enabling developers from many
```

Comment:
I didn't know the phrase `front and center` well till today. It's an adverb so I think it's better to use the phrase like this (or use another easier phrase for non-native speakers).

```suggestion
    JavaScript and HTML are front and center, enabling developers from many
```